app/code/Magento/Search/Test/Unit/Model/SearchEngine/ConfigTest.php

```diff
@@ -1,41 +1,46 @@
-<?php
+<?php declare(strict_types=1);
 /**
  * Copyright © Magento, Inc. All rights reserved.
  * See COPYING.txt for license details.
  */
 namespace Magento\Search\Test\Unit\Model\SearchEngine;
 
-class ConfigTest extends \PHPUnit\Framework\TestCase
+use Magento\Framework\TestFramework\Unit\Helper\ObjectManager;
+use Magento\Search\Model\SearchEngine\Config;
+use Magento\Search\Model\SearchEngine\Config\Data;
+use PHPUnit\Framework\MockObject\MockObject;
+use PHPUnit\Framework\TestCase;
+
+class ConfigTest extends TestCase
 {
-    /** @var \Magento\Search\Model\SearchEngine\Config\Data|\PHPUnit_Framework_MockObject_MockObject */
+    /** @var Data|MockObject */
     protected $dataStorageMock;
 
-    /** @var \Magento\Framework\TestFramework\Unit\Helper\ObjectManager */
+    /** @var ObjectManager */
     protected $objectManager;
 
-    protected function setUp()
+    protected function setUp(): void
     {
-        $this->dataStorage = $this->createMock(\Magento\Search\Model\SearchEngine\Config\Data::class);
```

Comment:
Most likely the test fails due to the missing `dataStorageMock`.

Reply:
PHPStan should detect that. But if not - I'm going to fix that either way. 
Thank you!